debian/changelog

```diff
@@ -1,3 +1,9 @@
+system76-driver (20.04.34~~alpha) focal; urgency=low
+
+  * Daily WIP for 20.04.34
```

Comment:
Add a message after this line, `meer5: Fix audio on revision without headphone jack`

Reply:
Sorry, that's what the force push was supposed to do, but I'd forgotten to add before committing.